### ISSUE TITLE
Pain worse when lying flat

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10181]
**Preferred term label:**
Pain worse when lying flat

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pericarditis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000014
- label: Pain worse when lying flat
- parents: HP:0025285
- pmid: PMID:31941732
- def: Refers to pain that is worse when the affected individual lies down (flat).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10181
- synonyms: Pain worse when lying down(NONE)
